cmd/lncli/commands.go

```diff
@@ -1401,6 +1401,101 @@ func create(ctx *cli.Context) error {
 	client, cleanUp := getWalletUnlockerClient(ctx)
 	defer cleanUp()
 
+	// We'll check to see if the user provided any static channel backups (SCB),
```

Comment:
The comment looks a bit lost up here. I know it is intended to describe the next few blocks.
I think it would better fit as the comment block for the switch statement below.

Reply:
Moving